src/wiki-table-editor.jsx

```diff
@@ -5,7 +5,24 @@ import * as Table from 'reactabular-table';
 import * as dnd from 'reactabular-dnd';
 import cloneDeep from 'lodash/cloneDeep';
 import findIndex from 'lodash/findIndex';
+import draggableRow from './draggable-row.jsx';
 
+const DndRow = draggableRow('tr');
+class DndCell extends React.Component {
+  render() {
+    let props = cloneDeep(this.props);
+    let canDragCell = props.canDragCell;
+    delete props.connectDragSource;
+    delete props.canDragCell;
+    const td = (<td {...props}>{this.props.children}</td>);
```

Comment:
It's the spread operator. You can also use it for arrays and objects.

Example: If `props = {a: 1, b: 2, c: 3}`, this will expand to:

```jsx
<td a="1", b="2", c="3">
```